src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerMapImageLayer/LocalServerMapImageLayer.xaml.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using ArcGISRuntime.Samples.Managers;
+using Esri.ArcGISRuntime.LocalServices;
+using Esri.ArcGISRuntime.Mapping;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace ArcGISRuntime.WPF.Samples.LocalServerMapImageLayer
+{
+    public partial class LocalServerMapImageLayer
+    {
+        // Hold a reference to the local map service
+        private LocalMapService _localMapService;
+
+        public LocalServerMapImageLayer()
+        {
+            InitializeComponent();
+
+            // Create the UI, setup the control references and execute initialization
```

Comment:
This comment seems to get copied to every sample, but isn't always accurate. In this case, no UI is being created, e.g.